common/src/main/java/co/sodalabs/view/ContextExtensions.kt

```diff
@@ -0,0 +1,22 @@
+package co.sodalabs.view
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import android.os.Build
+import android.support.v4.content.ContextCompat
+import android.support.v4.graphics.drawable.DrawableCompat
+
+fun Context.getBitmapFromVectorDrawable(drawableId: Int): Bitmap {
```

Comment:
Wait, doesn't Android auto convert to PNGs for you?

Reply:
Nope. Not vector drawables.